torch/_dynamo/skipfiles.py

```diff
@@ -196,6 +196,11 @@ def _module_dir(m: types.ModuleType):
     _module_dir(torch) + "_export/wrappers.py",
 }
 
+if torch.distributed.is_available():
```

Comment:
config this too, I guess.

Reply:
Actually, I am not sure this matters, let's see if tests fail, the inline vs allow refactor has made this a little more confusing.